lua/packer.lua

```diff
@@ -188,13 +291,19 @@ manage = function(plugin)
     end
   end
 
-  plugin.install_path = util.join_paths(plugin.opt and config.opt_dir or config.start_dir,
```

Comment:
This chunk maybe looks more complicated than it actually is—All I'm doing here is to ask, "has this already been `use`d before? If not, do <whatever packer did before>. If we've seen it before, skip the `setup` stuff (since it's already been done the first time around) and merge the new spec into the old one".